includes/class-wc-subscriptions-email.php

```diff
@@ -306,11 +317,17 @@ public static function order_details( $order, $sent_to_admin = false, $plain_tex
 	public static function subscription_details( $subscriptions, $order = null, $sent_to_admin = false, $plain_text = false, $skip_my_account_link = false ) {
 		$template = ( $plain_text ) ? 'emails/plain/subscription-info.php' : 'emails/subscription-info.php';
 
+		if ( ! is_array( $subscriptions ) ) {
+			$subscriptions = [ $subscriptions ];
+		}
+
+		$order = ! $order && ! empty( $subscriptions ) ? reset( $subscriptions )->get_parent() : $order;
+
 		wc_get_template(
 			$template,
 			array(
-				'order'                => ! $order ? $subscription->get_parent() : $order,
-				'subscriptions'        => is_array( $subscriptions ) ? $subscriptions : [ $subscriptions ],
+				'order'                => $order ?? null,
```

Comment:
Isn't `$order ?? null` just `$order`?

Reply:
Yes, yes it is. Don't remember what I was thinking when I first did this 🤔 Thanks for catching this! Fixed in [2e83a79](https://github.com/Automattic/woocommerce-subscriptions-core/pull/756/commits/2e83a79ed321c149fab5ea5fdb59d2c776002b39)